tmpl/wiki/tutorial-lwt.md

```diff
@@ -74,508 +111,327 @@ Assuming you are building for POSIX using native kernel sockets, compile the app
 ```
 
 If you are building for a different MirageOS platform, change the `--unix` switches appropriately (presently either `--xen` or `--unix`).
-The `OS` module packs several sub-modules depending of the backend, [unix](https://github.com/mirage/mirage-platform/tree/master/unix/lib)
+The `OS` module packs several sub-modules depending on the backend, [unix](https://github.com/mirage/mirage-platform/tree/master/unix/lib)
 or [xen](https://github.com/mirage/mirage-platform/tree/master/xen/lib).
 
 ###Solution
 
 ```
   open V1_LWT (* provides the CONSOLE signature *)
-  open Lwt    (* provides bind and join *)
+  open Lwt    (* provides >>= and join *)
```

Comment:
wondering whether we should propose in this tutorial to only `open Lwt.Infix`, instead of the big `Lwt`?